plugins/interactive/shellhandler.go

```diff
@@ -354,7 +354,7 @@ func (s *ShellHandler) sendAcks() {
 
 		// Send an acknowledgment message (this is for congestion control)
 		binary.BigEndian.PutUint32(ack[2:], uint32(N))
-		s.send(ack)
+		s.send(ack, true)
```

Comment:
If a program exists before stdin is closed, then we can't acknowledge the rest of stdin... but that's okay...